content/articles/top-container-orchestration-tools/index.md

```diff
@@ -0,0 +1,137 @@
+
+If you have applications to develop, deploy and run, containers come in handy. With containers, applications are moved seamlessly between servers. Therefore, you need a management system that automatically runs containers, shuts them down when needed, and controls how containers access resources. That is where orchestration tools come in.
+
+Container orchestration tools enable developers to manage and automate containerized application tasks. These tasks include linking containers, running instances, provisioning hosts, and rescheduling failed containers.
+
+Container orchestration tools help in extending the lifecycle of applications, ease deployment, and manage application configurations. These tools also improve container security by setting limits while accessing containers.
+
+This article will introduce you to container orchestration tools. Also, it will give you an overview of container orchestration and why it is important.
+
+### Table of Contents
+- [Introduction to container orchestration](#introduction-to-container-orchestration)
+- [Why container orchestration is important](#why-container-orchestration-is-important)
+- [Top container orchestration tools](#top-container-orchestration-tools)
+
+### Introduction to container orchestration
+To understand the problem solved by [container orchestration](https://docs.docker.com/get-started/orchestration/). Think of a case where you have two containers:
+- A front-end application container
+- A back-end application container
+
+These two containers need to work together as one. The solution to the above problem is container orchestration.
+
+Container orchestration is the automation of all processes of organizing and managing containers. Container orchestration automates deploying, scheduling, management, and networking of containers. Organizations that deploy, manage and host containers should use container orchestration. It helps you to deploy the application in different environments seamlessly.
+
+Microservices in containers make it straightforward to orchestrate services such as scalability, security, storage, flexibility, and networking. Containers help us combine applications to development, deployment, management, and running. At the same time, we need a set of tools to manage these containers. This is where container orchestration tools come in handy.
```

Comment:
Add 1 -2 lines explanation about what microservices are, and how they're used in containers